x-pack/plugins/files/server/blob_storage_service/adapters/es/content_stream/content_stream.ts

```diff
@@ -0,0 +1,338 @@
+/*
```

Comment:
This code was slightly adapted from the original ContentStream implementation given that the "head" file is not going contain special metadata. Instead this is stored in the new `file` Saved Object and passed through for reading/writing files.